homeassistant/components/tibber/sensor.py

```diff
@@ -450,7 +455,7 @@ def __init__(
     @property
     def available(self) -> bool:
         """Return True if entity is available."""
-        return self._tibber_home.rt_subscription_running
+        return cast(bool, self._tibber_home.rt_subscription_running)
```

Comment:
I do not really understand why I need to cast here?

https://github.com/Danielhiversen/pyTibber/blob/b29efdd84d399dfdf1be83344a3ee193769b0cc1/tibber/tibber_home.py#L432

Reply:
Should be in the package `sub_manager` isn't typed so `is_running` could come back as None and then the return statement could also return None (I believe).

Reply:
Ahh, that make sense.
sub_manager is not typed, then I guess it is fine to cast here? Or should I do a cast in pyTibber?

Reply:
Would probably be sufficient to change line https://github.com/Danielhiversen/pyTibber/blob/master/tibber/tibber_home.py#L436 into `and self._tibber_control.sub_manager.is_running is True`
You could also use bool() to force a boolean

Reply:
Hmm, was not able to make that work

Reply:
The upstream package is missing the PEP-561 marker. Add a `py.typed` file, and ensure it is packaged.

ref: <https://peps.python.org/pep-0561/>

Reply:
Thanks 👍